Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNext(_ viewController: OnboardingViewController)
+}
+
+class OnboardingViewController: UIViewController {
+
+    var stackView: UIStackView!
+    var subtitleLabel: UILabel!
+    
+    private var titleLabel: UILabel!
+    private var imageView: UIImageView!
+    private var nextButton: UIButton!
```

Comment:
can we make these `private` // `let`

Reply:
In the next PR we are changing the architecture of OnboardingViewController and OnboardingPageViewController that will allow these to be private